src/main/java/com/github/manolo8/darkbot/modules/utils/NpcAttacker.java

```diff
@@ -133,10 +136,12 @@ public double modifyRadius(double radius) {
         if (target.health.hpPercent() < 0.25 && target.npcInfo.extra.has(NpcExtra.AGGRESSIVE_FOLLOW)) radius *= 0.75;
         if (target != hero.target || !hero.isAttacking(target) || castingAbility()) radius = Math.min(550, radius);
         else if (!target.locationInfo.isMoving() || target.health.hpPercent() < 0.25) radius = Math.min(600, radius);
+        if (target.npcInfo.extra.has((NpcExtra.AGGRESSIVE_APPROACH)) && !closeRange) radius *= 0.55;
+        radius += bar.findItemById("ability_zephyr_mmt").map(i -> i.quantity).orElse(0d) * 5;
 
-        return radius + bar.findItemById("ability_zephyr_mmt").map(i -> i.quantity).orElse(0d) * 5;
+        if (main.hero.locationInfo.distance(target) < radius + 0.45) closeRange = true;
```

Comment:
This 0.45 is an absolute value, i'm pretty sure it's intended to be a multiplying factor
The issue is this may never be true, if you try to get to a distance of 600 with an npc, if the npc is moving away from you you will never get 600 units close.
When you change this `closeRange` variable may not be reliable at all as it is here